modules/gdscript/gdscript.cpp

```diff
@@ -2368,62 +2368,60 @@ void GDScriptLanguage::frame() {
 
 /* EDITOR FUNCTIONS */
 void GDScriptLanguage::get_reserved_words(List<String> *p_words) const {
-	// TODO: Add annotations here?
 	static const char *_reserved_words[] = {
-		// operators
+		// Control flow.
```

Comment:
Please, avoid this kind of reorder, since it does not improve anything and just make the PR artifically bigger and adds confusion on "why" it was done.
No need to fix it, just a thing to keep in mind for the future.

Reply:
There's no confusion IMO, it was done because the comments were used to categorize keywords, and multiple items were wrong

Reply:
Reodering for clarity seems good to me, but I want to make sure the logic used is consistent.

What's the logic used here for both the categories and the order of keywords within each category?
I see e.g. that `token_names` in `gdscript_tokenizer.cpp` has similar but not fully matching categories, and there the tokens are listed alphabetically within each category - like it used to be done here before this change.

If this is based on an enum somewhere, it should be referenced in a comment, ideally in both places so we know what to update whenever the enum changes.

Reply:
I sorted them alphabetically and added comments.